### PR TITLE
New package: msdfgen-1.7.1

### PR DESCRIPTION
**Pull request by vktec**

### CODE REVIEW

File: srcpkgs/msdfgen/patches/cmake-install.diff [hunk 1]
```diff
@@ -0,0 +1,274 @@
+commit 31dda2d47417b8999b9c873d642fba9a30cd2f9d
```
Comment (sgn) [inline srcpkgs/msdfgen/patches/cmake-install.diff:1]: Does the build system work without this patch?

Comment (vktec) [reply srcpkgs/msdfgen/patches/cmake-install.diff:1]: No, there is no install step in the old cmake file

Comment (sgn) [reply srcpkgs/msdfgen/patches/cmake-install.diff:1]: Manually install them then.

**Comment by ericonr** [comment]
Ping

**Comment by ericonr** [comment]
Closing due to inactivity, feel free to reopen.

**Pull request closed without merging**